{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from {{ cookiecutter.project_slug }}.core.models import User
+
+
+@pytest.fixture
+def user(db):
+    return User.objects.create(email="user@example.com", password="1234", first_name="test", last_name="user")
```

Comment:
The beginnings of a conftest. There are other fixtures we have that should get moved here, but I'll do that in a more testing focused PR.